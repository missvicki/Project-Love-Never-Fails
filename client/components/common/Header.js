import React from "react";
import { Menu, Image } from "semantic-ui-react";
import { NavLink } from "react-router-dom";

import logo from "../../assets/images/ilogo.png";
import AllMenu from "./AllMenu";
import "../../assets/styles/header.css";

const Header = () => (
  <div className="header">
    <Menu secondary className="mainMenu" stackable>
      <Menu.Item header>
        <NavLink to="/">
          <Image className="logo" src={logo} />
        </NavLink>
      </Menu.Item>
      <AllMenu items="navItems" />
    </Menu>
  </div>
);

export default Header;
