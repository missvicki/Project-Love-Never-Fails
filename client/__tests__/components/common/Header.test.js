import React from "react";
import { shallow } from "enzyme";
import Header from "../../../components/common/Header";

describe("<Header />", () => {
  const wrapper = shallow(<Header />);
  it("should render Header component without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
