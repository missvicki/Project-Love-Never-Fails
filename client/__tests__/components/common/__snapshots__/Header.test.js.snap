// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render Header component without crashing 1`] = `
<div
  className="header"
>
  <Menu
    className="mainMenu"
    secondary={true}
    stackable={true}
  >
    <MenuItem
      header={true}
    >
      <NavLink
        to="/"
      >
        <Image
          as="img"
          className="logo"
          src="test-file-stub"
          ui={true}
        />
      </NavLink>
    </MenuItem>
    <AllMenu
      items="navItems"
    />
  </Menu>
</div>
`;
